sycl/test/regression/atomic_load.cpp

```diff
@@ -0,0 +1,41 @@
+// RUN: %clangxx -fsycl %s -o %t.out -lOpenCL
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// RUN: %clangxx --sycl -S -x c++ -O0 -emit-llvm %s -o - | FileCheck %s
+#include <CL/sycl.hpp>
+using namespace cl::sycl;
+
+template <typename T>
+class foo;
+
+template<typename T>
+void kernel_func(T val) {
+  cl::sycl::queue testQueue;
+
+  T data = val;
+  buffer<T,1> buf(&data, range<1>(1));
+
+  testQueue.submit([&](cl::sycl::handler &cgh) {
```

Comment:
This is illogical after `using namespace cl::sycl` at line 8. I'd suggest either removing `using` directive or `cl::sycl` specialization in the code. AFAIK, 1 is the preferred option.

Reply:
My mistake. I prefer to remove `cl::sycl` specialization as it will simplify test readability

Reply:
OK. I think it doesn't matter much for tests.